src/views/HomePage/HomePage.tsx

```diff
@@ -2,19 +2,42 @@ import { TitleBanner } from '@ui/components/TitleBanner';
 import StyledHomePage from './StyledHomePage';
 import logo from '/wwtbtfc.svg';
 import { NavigationButton } from '@ui/components/NavigationButton';
+import colors from '@ui/styles/colors';
 
 /**
  * Main home page for the app
  */
 export const HomePage = () => {
+  const oneLiners = [
+    'because affection is a zero sum game',
+    "because there's not enough tension in this family",
+    'because it was getting too chummy around here',
+    'because mama raised exactly one loser',
+    "because there's not enough room in this house for the both of us",
+  ];
   return (
     <>
```

Comment:
You don't need this fragment since everything is included inside the `<StyledHomePage />` component.

```suggestion
```